Jenkinsfile

```
@@ -1,5 +1,5 @@
 buildPlugin(useAci: true, configurations: [
-    [platform: 'linux', jdk: '8'],
-    [platform: 'windows', jdk: '8'],
-    [platform: 'linux', jdk: '11']
+    [platform: 'linux', jdk: '8', jenkins: null],
+    [platform: 'windows', jdk: '8', jenkins: null],
```

nit: `jenkins: null` is the default, so you can leave it out if desired:

```suggestion
    [platform: 'linux', jdk: '8'],
    [platform: 'windows', jdk: '8'],
```